pkg/client/clientset/versioned/typed/noderecovery/v1alpha1/noderecovery_client.go

```diff
@@ -37,8 +37,8 @@ type NoderecoveryV1alpha1Client struct {
 	restClient rest.Interface
 }
 
-func (c *NoderecoveryV1alpha1Client) NodeRemediations(namespace string) NodeRemediationInterface {
-	return newNodeRemediations(c, namespace)
+func (c *NoderecoveryV1alpha1Client) NodeRemediations() NodeRemediationInterface {
```

Comment:
Because nodes have no namespace?

Reply:
Correct, nodes have cluster scope, so it pretty logical that node remediation object also will have cluster scope.